### ISSUE TITLE
Number of WAL files for a tserver can become large

### ISSUE THREAD

**Issue by adamjshook** [apache/accumulo#1336]
I've been experiencing a case where TabletServers will hold on to many more WAL logs than normal in a (seemingly) unbounded fashion until it is restarted.  As a byproduct, it causes extremely long recovery periods after the restart.  TabletServers will have hundreds or thousands of WAL entries in ZooKeeper instead of a normal 3-5.

Accumulo Version 1.9.3

**Comment by ctubbsii** [comment]
@keith-turner , I noticed you self-assigned this about a month ago. Are you still looking into this? Have you made any progress?

**Comment by DomGarguilo** [comment]
Does this issue exist in any version later than the reported 1.9.3?

**Comment by amcdonaldccri** [comment]
Yes, unfortunately it happened in our prod environment using accumulo 2.1.3/hadoop 3.2.2 once about two weeks ago.  In our prod accumulo 2.0.0 cluster it happens more often.
